src/ShowCustomerData.tsx

```diff
@@ -49,22 +56,20 @@ export const ShowCustomerData: React.FC<Props> = ({onSubmit}) => {
         fetchData()
     }, []);
 
-    useEffect(() => {
-        restClient.get<ChuckNorrisJokeResponse>(
-            'https://api.chucknorris.io/jokes/random'
-        ).then(joke => setJoke(joke.value))
-    }, []);
-
     const submit = (text: string, checked: boolean) => {
         setValidInput(true);
-        if (text.length >= 0 && text.length < 3) {
+        if (text.length >= 0 && text.length < 3 || age === undefined) {
```

Comment:
Here we could show a proper message or a generic one. Now we are saying "test length not valid" but that's not correct